features/steps/fhir.py

```diff
@@ -144,7 +145,8 @@ def step_impl(context):
 @given('there is at least 1 entry')
 def step_impl(context):
     resource = context.response.json()
-    entries = resource.get('entry', [])
+    entries = [entry['resource'] for entry in resource.get('entry', [])
```

Comment:
Hmm, that makes sense. Should I only update the two tests I've got here to do that? And the logic they use to know what resources to filter out... Should I change the test to `there is at least 1 {resource_type} entry`?